projects/packages/connection-ui/.gitattributes

```
@@ -0,0 +1,15 @@
+# Files not needed to be distributed in the package.
+.gitattributes    export-ignore
+package.json      export-ignore
+webpack.config.js export-ignore
+gulpfile.babel.js export-ignore
+.babelrc          export-ignore
+
+# Files to include in the mirror repo
+/build/index.asset.php  production-include
+/build/index.js         production-include
+
+# Files to exclude from the mirror repo
+.gitignore        production-exclude
+_inc/             production-exclude
```

This tags the directory, but not the files inside it. So they still get included.
```suggestion
_inc/**           production-exclude
```